R/checks.R

```diff
@@ -25,8 +25,72 @@ copy_no_clobber = function(from_file,to_file,force=FALSE){
     file.copy(from_file,to_file)
   }
 }
+#' Check for a remote session and automagically confirm setup will work properly
+#' 
+#' This function determines if a user is working in GAMBLR remotely and, if so, will
+#' check if their config is loaded properly and ssh_session is available. 
+#'
+#' @param auto_connect Set to TRUE to ensure an ssh_session is created if absent
+#'
+#' @return
+#' @export
+#'
+#' @examples
+check_remote_configuration = function(auto_connect=FALSE){
```

Comment:
See PR review comment for more thoughts on this. In short, if I specify `Sys.setenv(R_CONFIG_ACTIVE= "remote")` the function stops at line 48. But if I specify it as such, `R_CONFIG_ACTIVE =  "remote"`, the function seems to work and I can proceed with other functions that internally call this, e.g `get_gambl_metadata.`.